exercises/reverse-string/example.rs

```diff
@@ -1,5 +1,3 @@
 pub fn reverse(input: &str) -> String {
-    let mut output = String::with_capacity(input.len());
-    output.extend(input.chars().rev());
-    output
+    input.chars().rev().collect::<String>()
```

Comment:
The `::<String>` turbofish should not be necessary

Reply:
I'm aware it's not "necessary"; I feel it's best to teach students from the beginning to be explicit.

Reply:
Makes sense,  but I suspect that not many student would (or even should) refer to this repo in order to find the solution.
Besides if the automatic mentoring is ever introduced to the Rust track, I think the example solution would be a go to example for the automatic tool, therefore it should contain as little elements as possible

Reply:
The return type of the function is explicit. I won't say that it's better to let the type inferencer do its job implicitly in _every_ case where it's capable of doing so, as there are times when making things explicit makes it easier to understand what's going on. I will say that in this case, things seem clear enough to me that I'd prefer to leave things implicit.